src/plugins/numberline/components/numberline-tile.tsx

```diff
@@ -0,0 +1,347 @@
+
+      outerPoints.enter()
+        .append("circle").attr("class", "outer-point")
+        .attr('cx', (p) => xScale(p.currentXValue ?? numberlineDomainMin)) //mapped to axis width
+        .attr('cy', yMidPoint).attr('r', outerPointRadius).attr('id', p => p.id)
+        .classed("point-outer-circle", true)
+        .call((e) => handleDrag(e)); // Attach drag behavior to newly created circles
+
+      // --- Update functions outer hover circles
+      outerPoints
+        .attr('cx', (p) => xScale(p.currentXValue ?? numberlineDomainMin)) //mapped to axis width
+        .classed("hovered", (p, idx) => (hoverPointId === p.id))
+        .call((e) => handleDrag(e)); // pass again in case axisWidth changes
+
+      outerPoints.exit().remove(); //cleanup
+
+      /* =========================== [ Inner Circles ] ============================= */
+      //---- Initialize inner hover circles
+      const innerPoints = svg.selectAll<SVGCircleElement, PointObjectModelType>('.circle,.inner-point')
+        .data(content.pointsArr);
+
+      // Initialize Attributes
+      innerPoints.enter()
+        .append("circle")
+        .attr("class", "inner-point")
+        .attr('cx', (p) => xScale(p.xValue || numberlineDomainMin)) //mapped to axis width
+        .attr('cy', yMidPoint).attr('r', innerPointRadius).attr('id', p => p.id)
+        .classed("point-inner-circle", true)
+        .call((e) => handleDrag(e)); // Attach drag behavior to newly created circles
+
+      // --- Update functions inner circles
+      innerPoints
+        .attr('cx', (p, idx) => xScale(p.currentXValue || numberlineDomainMin))
+        .classed("selected", (p)=> p.id in content.selectedPoints)
+        .call((e) => handleDrag(e)); // pass again in case axisWidth changes
+
+      innerPoints.exit().remove(); //cleanup
+    };
+    updateCircles();
+  }
+
+  // Set up toolbar props
+  const toolbarProps = useToolbarTileApi({ id: model.id, enabled: !readOnly, onRegisterTileApi, onUnregisterTileApi });
+
+  return (
+    <div
+      className={classNames("numberline-wrapper", { "read-only": readOnly })}
+      onKeyDown={(e) => hotKeys.current.dispatch(e)}
+      tabIndex={0}
+    >
+      <div className={"numberline-title"}>
+        <BasicEditableTileTitle
+          model={model}
+          readOnly={readOnly}
+          scale={scale}
+        />
+      </div>
+      <NumberlineToolbar
+        documentContent={documentContent}
+        tileElt={tileElt}
+        {...toolbarProps}
+        scale={scale}
+        handleClearPoints={() => content.deleteAllPoints()}
+        handleDeletePoint={deleteSelectedPoints}
+      />
+      <div
+        className="numberline-tool"
+        ref={documentScrollerRef}
+        data-testid="numberline-tool"
+        style={{"height": `${kNumberLineContainerHeight}`}}
+      >
+        <div className="numberline-tool-container" >
+          <i className="arrow left" style={{ left: xShiftNum + kArrowheadOffset, top: kArrowheadTop }}/>
```

Comment:
We noticed (and is also in main) - the left and right arrow bounds on the numberline are above everything else (points on the numberline). 

Could you try adding a z-index on the` .right` and `.left `CSS classes in` numberline-tile.scss` and see if that works? 

If not then maybe check with Leslie to see we want to write a chore ticket to fix this.